README.md

```diff
@@ -10,7 +10,15 @@ We want to enable users to mount secrets on a workload (deployment/statefulset)
 
 Once the application is running, user can "sneak-in" (create) secret as a VolumeMount automatically along with a deployment creation.
 
-Just update your secrets as a key:value pair at path /mnt/data/secret-sneaker-data/secrets.json on your host machine (Create the entire path if required)
+Just update your secrets as a key:value pair at path /mnt/data/secret-sneaker-data/secrets.json on your host machine (Create the entire path if required) as shown in sample JSON below;
```

Comment:
1. why do I have to create the json?
2. Why at the specific location?

also 
```
Sneak secret file in your namespace automatically while creating a deployment
```
this didn't make sense to me.